modules/local/multifasta_to_csv.nf

```diff
@@ -15,16 +15,15 @@ process MULTIFASTA_TO_CSV {
 
     script:
     """
-    echo "id,sequence" >> input.csv
-    echo -e ${seq_name.sequence},`awk -F ' ' '!/^>/ {print \$0}' ${fasta} | tr "\n" ":" | awk '{gsub(/:\$/,""); print}'` >> input.csv
+    echo -e id,sequence'\\n'${seq_name.sequence},`awk '!/^>/ {print \$0}' ${fasta} | tr '\\n' ':' | sed 's/:\$//'` > input.csv
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         sed: \$(echo \$(sed --version 2>&1) | sed 's/^.*GNU sed) //; s/ .*\$//')
-        gawk: \$(awk -Wversion 2>/dev/null | sed 's/.*wk //; s/,.*\$//')
```

Comment:
Why remove the `gawk` version?

Reply:
I did so while debugging, because I had some issues with this process related to escaping or not characters with the old command and it was crashing at that stage. But after updating to the new echo command, it should work. So, it's up  to you whether to include it or not